src/rust-crypto/verification.ts

```diff
@@ -700,3 +702,31 @@ export function verificationMethodIdentifierToMethod(method: string): RustSdkCry
     }
     return meth;
 }
+
+/**
+ * Return true if the event is a verification event
```

Comment:
Let's be a bit more specific about what we are matching here.

```suggestion
 * Return true if the event's type matches that of an in-room verification event
```